.github/workflows/main.yaml

```diff
@@ -242,7 +242,7 @@ jobs:
         run: |
           command -v ginkgo || go install github.com/onsi/ginkgo/v2/ginkgo@${{ env.GINKGO_VERSION }}
           go install sigs.k8s.io/kind@v0.11.1
```

Comment:
kind version also need updating
ref: https://github.com/kubernetes-sigs/kind/releases
maybe we should specify the kindest/node image tag when we call `kind create cluster` to use v1.23.15 k8s

Reply:
it seems kindest/node:v1.23.15 not exist 👀 
the closest is kindest/node:v1.23.13

Reply:
`kindest/node:v1.23.13` is also fine :)